config.json

```diff
@@ -381,6 +381,18 @@
         "sorting",
         "trees"
       ]
+    },
+    {
+      "slug": "nth-prime",
+      "uuid": "b470633d-6d2c-4245-92eb-37ea80500f2b",
+      "core": false,
+      "unlocked_by": "binary-search-tree",
```

Comment:
Please change this to be unlocked by armstrong-numbers. I'll update the other feedback for the other PRs to ignore the request of having the last core exercise as the unlocked_by

Reply:
@Stargator Updated